mayastor/tests/nexus_multipath.rs

```diff
@@ -262,14 +278,142 @@ async fn nexus_multipath() {
         "should have default registered key"
     );
 
-    let output_dis2 = Command::new("nvme")
-        .args(&["disconnect"])
-        .args(&["-n", &rep_nqn])
-        .output()
+    nvme_disconnect_nqn(&rep_nqn);
+}
+
+#[tokio::test]
+/// Create a nexus with a remote replica on 1 node as its child.
+/// Create another nexus with the same remote replica as its child, verifying
+/// that the write exclusive reservation has been acquired by the new nexus.
+async fn nexus_resv_acquire() {
+    let test = Builder::new()
+        .name("nexus_resv_acquire_test")
+        .network("10.1.0.0/16")
+        .add_container("ms2")
+        .add_container("ms1")
+        .with_clean(true)
+        .build()
+        .await
         .unwrap();
-    assert!(
-        output_dis2.status.success(),
-        "failed to disconnect from remote replica, {}",
-        output_dis2.status
+
+    let mut hdls = test.grpc_handles().await.unwrap();
+
+    // create a pool on remote node 1
+    hdls[0]
```

Comment:
Its not certain that this is "node1" -- the grpc_handles() function may return a different order. If you need to be sure it's node1 (as in the first ms instance)  -- it's better to get a handle by name. Do you know for sure the order here does not matter?

Reply:
I hadn't realised that. The node order indeed doesn't matter, as long as they are referred to consistently. I'll clarify it one way or another, either by getting the handle by name or in the comments.